extensions/punto/src/index.ts

```diff
@@ -66,11 +73,11 @@ function detectLayout(input: string): string {
 function switchCharacterLayout(char: string): string {
   if (en_ru.has(char)) {
     // console.log(char + " detected in en dict")
-    switchLayout("Russian");
+    // switchLayout("Russian");
     return en_ru.get(char) ?? char;
   } else {
     // console.log(char + " is probably detected in ru dict")
-    switchLayout("ABC");
+    // switchLayout("ABC");
     return ru_en.get(char) ?? char;
```

Comment:
If you want to support an additional layout, you need to create a separate mapping